tests/test_pg_conn.py

```diff
@@ -0,0 +1,34 @@
+import psycopg as pg
+from musicbrainz_bot import config
+import pytest
+
+
+def make_connection(use_test_db: bool) -> bool:
+    if use_test_db is True:
+        URI = config.MB_TEST_DB
+    else:
+        URI = config.MB_DB
+
+    try:
+        conn = pg.connect(URI)
+        conn.close()
+        return True
+
+    except Exception:
+        return False
+
+
+def test_TEST_DB_connection() -> None:
```

Comment:
is it helpful to have explicit tests for the DB connection?  presumably all the rest of the tests would already fail if the connection didn't work? :)  (maybe I'm missing the point)

Reply:
Yea, I had that one setup kind of like a scaffolding to help me fix some of the docker issues actually.

Would you like to have this particular test removed?

Reply:
If your Docker issues are resolved, let's remove these, since they don't test any of the actual bot code.